### DISCUSSION TITLE
Possible to add reaction on individual thread comments

### DISCUSSION THREAD

**Discussion by ldelossa** [pwntester/octo.nvim#273] [Q&A] [answered]
is it possible to add reaction like thumbs up to individual thread comments? When I try this in a PR (my own pr, someone else reviewed it and I'm implementing their changes) it only will add a reaction to the PR itself.

**Comment by williambdean** [comment, marked as answer]
Hey @ldelossa, I think that this is resolved, right? Using `Octo reaction <reaction>` should work when the cursor is on comment or review thread.
